### ISSUE TITLE
PHP Fatal Error 'yii\base\ErrorException' with message 'Class 'craft\gql\base\Generator' not found'

### ISSUE THREAD

**Issue by ademers** [solspace/craft3-calendar#45]
**Describe the bug or issue you're experiencing**

When attempting to update from 3.0.15 to 3.3.1 on Craft 3.5.19.1 after running `./craft migrate/all` command:

```
vagrant@homestead:~/sites/example$ ./craft migrate/all
PHP Fatal Error 'yii\base\ErrorException' with message 'Class 'craft\gql\base\Generator' not found'

in /home/vagrant/sites/example/vendor/solspace/craft3-calendar/packages/plugin/src/Bundles/GraphQL/Types/Generators/AbstractGenerator.php:11

Stack trace:
#0 [internal function]: yii\base\ErrorHandler->handleFatalError()
#1 {main}
```

**Steps to reproduce**

1. Set Calendar version to 3.3.1 in `composer.json`
2. Run `composer update`
3. Run `./craft migrate/all`
4. See error shown above

**Expected behavior**

No errors after running `./craft migrate/all`.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.5.19.1 (upgrading to 3.6.x currently not an option)
- Calendar Version: 3.3.1
- Calendar Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Additional context**

**Update**: In the meantime, I successfully updated to 3.2.1.

**Comment by gustavs-gutmanis** [comment]
@ademers it looks like the `Generator` class has been introduced with Craft `v3.6.0`, is it possible for you to upgrade to Craft `3.6.0+`?

**Comment by ademers** [comment]
@gustavs-gutmanis I was planning on upgrading to Craft 3.6.x, but then the 3.7 beta was released. To avoid confusing the content authors, I've decided that it's best to upgrade from 3.5.x to 3.7.x directly and skip 3.6.x. I can wait until then (August) to update Calendar. Unless I missed it, I didn't see a notice in the Craft CP or Calendar changelog that Calendar 3.3.x requires Craft 3.6.x+. Maybe add this? Thanks!
